frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/link_preview/custom_link_preview.dart

```diff
@@ -1,38 +1,50 @@
+import 'package:appflowy/generated/flowy_svgs.g.dart';
+import 'package:appflowy/generated/locale_keys.g.dart';
+import 'package:appflowy/mobile/presentation/widgets/widgets.dart';
+import 'package:appflowy/plugins/document/presentation/editor_plugins/actions/mobile_block_action_buttons.dart';
+import 'package:appflowy/plugins/document/presentation/editor_plugins/link_preview/shared.dart';
 import 'package:appflowy/shared/appflowy_network_image.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:easy_localization/easy_localization.dart';
 import 'package:flowy_infra_ui/flowy_infra_ui.dart';
 import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:provider/provider.dart';
 import 'package:url_launcher/url_launcher_string.dart';
 
 class CustomLinkPreviewWidget extends StatelessWidget {
   const CustomLinkPreviewWidget({
     super.key,
+    required this.node,
     required this.url,
     this.title,
     this.description,
     this.imageUrl,
   });
 
+  final Node node;
   final String? title;
   final String? description;
   final String? imageUrl;
   final String url;
 
   @override
   Widget build(BuildContext context) {
-    return InkWell(
-      onTap: () => launchUrlString(url),
-      child: Container(
-        clipBehavior: Clip.hardEdge,
-        decoration: BoxDecoration(
-          border: Border.all(
-            color: Theme.of(context).colorScheme.onSurface,
-          ),
-          borderRadius: BorderRadius.circular(
-            6.0,
-          ),
+    final fontSize = PlatformExtension.isDesktopOrWeb ? 16.0 : 14.0;
+    final width = PlatformExtension.isDesktopOrWeb ? 180.0 : 120.0;
```

Comment:
```suggestion
    final (fontSize, width) = PlatformExtension.isDesktopOrWeb ? (16.0, 180.0) : (14.0, 120.0);
```

Reply:
Should the fontSize on Desktop follow the FontSize from Document Appearance?

Reply:
Yes. Let me change it.